processingreport.qmd

```diff
@@ -2,32 +2,32 @@
 title: "U.S. Greenhouse Gas Center: Processing and Verification Reports"
 ---
 
-Welcome to the [U.S. Greenhouse Gas (GHG) Center: Processing and Verification Reports](https://earth.gov/ghgcenter), your gateway to exploring and analyzing curated datasets on greenhouse gas emissions.
+Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) processing and verification reports, your gateway to exploring and analyzing curated datasets on greenhouse gas emissions.
```

Comment:
Recommend replacing "Welcome to the [U.S. Greenhouse Gas (GHG) Center](https://earth.gov/ghgcenter) processing and verification reports, your gateway to exploring and analyzing curated datasets on greenhouse gas emissions. Delve deeper into each dataset's journey with our comprehensive processing and verification reports. Learn how we handle and verify data with care, ensuring accuracy and reliability every step of the way. Explore examples of these reports, providing transparency and insight into our meticulous processes." with:

Welcome to the U.S. Greenhouse Gas (GHG) Center processing and verification reports. These reports verify that the accuracy and integrity of each dataset in the US GHG Center is maintained once it is processed into the Center. 

The reports are grouped topically and labeled by dataset name. The dataset product type (model output, satellite observation, etc.) is noted next to each. Click on a dataset name to view the processing and verification report for that dataset.

Examples of processing that may occur include transforming data from its source format into a could-optimized format, converting the units of the source data into a more common or standard unit, and flagging "nodata" values to ensure accurate data visualization. We strive to handle all data with extreme care, and share these reports to provide transparency and insight into any processing that is applied, while ensuring accuracy and reliability every step of the way.